.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Algo não está funcionando/Something is not working
+about: Eu encontrei algum bug na aplicação/I found a bug in the app
+title: ''
+labels: ''
```

Comment:
Couldn't we already add the related tags?

Reply:
I though about not adding the labels because the issue may not be a bug, we need to evaluate it before tagging

Reply:
Then add a `Needs triage`? So we can actually be actionable about it?